tools/cpp/osx_cc_wrapper.sh.tpl

```diff
@@ -13,94 +13,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
-# OS X relpath is not really working. This is a wrapper script around gcc
-# to simulate relpath behavior.
-#
-# This wrapper uses install_name_tool to replace all paths in the binary
-# (bazel-out/.../path/to/original/library.so) by the paths relative to
-# the binary. It parses the command line to behave as rpath is supposed
-# to work.
-#
-# See https://blogs.oracle.com/dipol/entry/dynamic_libraries_rpath_and_mac
-# on how to set those paths for Mach-O binaries.
-#
-set -eu
 
-INSTALL_NAME_TOOL="/usr/bin/install_name_tool"
+# TODO(yannic): Remove this wrapper.
```

Comment:
FWIW when we remove these I think we should point these actions at wrapped_clang https://github.com/bazelbuild/bazel/pull/12356

Reply:
+1